Cleanup

tafuri opened the pull request:
Couple of changes to make the project easier to maintain and build.
Had to use VS syntax for VSINSTALLDIR to make it compile on my machine.

tafuri commented on drpdb/src/DIACommon.h:
```diff
@@ -24,20 +26,26 @@ namespace DIA2
+
+		const ULONG ChunkSize = 65536;
+		ULONG totalNumFetched{ 0 };
+		while(totalNumFetched < (ULONG)num)
```
Yes, you are right. Not sure how I managed to miss that :)

briterator commented:
I resolved the conflicts in #7 but the program now crashes at runtime, unable to locate the .dll files. Did you add a manifest or some other way to look up the dlls? They were deleted in fea6c18.

tafuri commented:
Sorry about that, I forgot to copy libmysql.dll to the output folder (I'm currently only using the tool with sqlite).
I fixed this and also added a runtime check for the library so the application will not crash if it is missing.
